src/DIRAC/WorkloadManagementSystem/Agent/SiteDirector.py

```diff
@@ -1470,3 +1047,64 @@ def sendPilotSubmissionMonitoring(self, siteName, ceName, queueName, numTotal, n
             return S_ERROR()
         self.log.verbose("Done committing to monitoring")
         return S_OK()
+
+
+class SubmissionPolicy(ABC):
```

Comment:
Just a personal preference, but I would move this out of this module.

Reply:
I moved that to the `Utilities` directory.